### ISSUE TITLE
No simblee connection with Android

### ISSUE THREAD

**Issue by TKA1988** [magicboxlabs/REFLO#2]
Here is the report from Android user.
---------------------------------------------------------------------------------------------------
The REFLO powers up and stabilizes at the preset initial temperature. I can see and Bluetooth pair with the REFLO from the phone and from a computer. So the REFLO does appear to be working correctly.

However the Simblee app (Android) does not seem to connect. It prints the lines "Found Simblees" and "Select Simblee" regardless of whether the REFLO is turned on or not. No other display or interaction appears possible. I've also uninstalled and reinstalled the app, making sure it has been given all requested permissions.

It looks as though my Android 8.0.0 phone (Sony Xperia XZ) is not supported. It does connect with my Galaxy S tablet (Android 6.0.1) but the app doesn't work correctly (repeatedly disconnects, can't control temperature in Manual mode).

It would seem I'm stuck until Simblee provides Android 8.0 support. Could it be possible to communicate with the controller via a BLE serial terminal session or directly through the USB connector? Seems unfortunate to be this close!

**Comment by TKA1988** [comment]
The software for Android was updated. Please check the following.
https://github.com/magicboxlabs/REFLO/tree/master/software

**Comment by mignon-p** [comment]
> The software for Android was updated. Please check the following.
> https://github.com/magicboxlabs/REFLO/tree/master/software

So, to be clear, I need to recompile the software and upload it to the REFLO using the USB connector on the circuit board?

**Comment by TKA1988** [comment]
Yes, you need to do. And don't forget to power supply from DC converter to the PCB when you upload it.

**Comment by jdimpson** [comment]
I need more help than this. I can get simblee for mobile on Android to connect and start a bake, but after that the app will not stay connected.

Are you saying the fix is to change the Arduino code on the reflo?  Where are the instructions for this? The simblee.com link provided in REFLO/software folder is broken.

Does this mean that I need different Arduino code on the REFLO depending on whether I control via Apple or Android? This simblee stuff is terrible.

What do you mean "don't forget to power supply from DC converter to the PCB when you upload it." Do you mean to provide DC power to the internal board? How? How do I upload the firmware? I assume it had a serial or usb header, but where can I see the documentation for that?

**Comment by TKA1988** [comment]
@jdimpson Yes, you need to change the Arduino code on the PCB by micro USB. I attached quick start guide for Simblee IDE installation. When you upload the code, you have to turn on power switch of REFLO so that it supplies 5V to the PCB. The code for Android is [here.]  (https://github.com/magicboxlabs/REFLO/tree/master/software)
If it is not going to work, we could send you the PCB optimized for Android.
We are sorry for inconvenience. Please keep us updated on this matter.
[Simblee_Quickstart_Guide_v1.0.pdf](https://github.com/magicboxlabs/REFLO/files/2729275/Simblee_Quickstart_Guide_v1.0.pdf)
